README.md

```diff
@@ -153,6 +153,15 @@ make full   ## or `make all`
 Run `make help` to see other make targets and the various command-line options.
 
 
+## Using the Limine bootloader instead of GRUB
+To use Limine instead of GRUB, clone pre-built limine and pass `bootloader=limine-iso` to make:
+```
+git clone https://github.com/limine-bootloader/limine.git --branch=v3.6-binary --depth=1
```

Comment:
I don't think this is futureproof, as the branch `v3.6-binary` could be removed at any time. Let's use a specific commit instead.

Also, please add the language tag to the top of this code block:
```
```sh
```